README.th.md

```diff
@@ -0,0 +1,42 @@
+- [English](README.md)
+- [日本語](README.ja.md)
+- [한국어](README.ko.md)
+- [Bahasa Indonesia](README.in.md)
+- [Español](README.es.md)
+- [Deutsch](README.de.md)
+- [français](README.fr.md)
+- [العربية](README.ar.md)
+- [Português Brasileiro](README.pt-br.md)
+- [Italiano](README.it.md)
+- [Nederlands](README.nl.md)
+- [Thai](README.th.md)
+- [简体中文](README.zh_hans.md)
+- [繁體中文](README.zh_hant.md)
```

Comment:
As per #37, we will only keep the list of all languages on the main English readMe, so that every readMe doesn't have to be updated each time a new language is added. Feel free to remove it here.

Reply:
Okay, I removed other readme links from my translation except for the English one.